redbeat/schedulers.py

```diff
@@ -455,9 +455,9 @@ def maybe_due(self, entry, **kwargs):
             try:
                 result = self.apply_async(entry, **kwargs)
             except Exception as exc:
-                logger.exception('Message Error: %s', exc)
+                logger.exception('Scheduler: Message Error: %s', exc)
```

Comment:
should this be lower case "scheduler"? Feels more consistent to me, not sure if this follows a pattern elsewhere.

Reply:
It follows the same pattern the original scheduler uses (lowercase `beat`, uppercase `Scheduler` don't ask me why): 
https://github.com/celery/celery/blob/7c3da03a07882ca86b801ad78dd509a67cba60af/celery/beat.py#L267
https://github.com/celery/celery/blob/7c3da03a07882ca86b801ad78dd509a67cba60af/celery/beat.py#L616

Reply:
🤷 ok